engine/src/main/java/org/terasology/rendering/nui/animation/AbstractAnimation.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2014 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.animation;
+
+import java.util.List;
+import java.util.ArrayList;
+
+/*
+ */
+public abstract class AbstractAnimation {
+    private List<AnimationListener> myListeners;
```

Comment:
myCamelCase is a valid field convention, but isn't used at all in Terasology - I'd stick to this.camelCase instead